demo_classification_tf.py

```diff
@@ -0,0 +1,68 @@
+import sys
+import tensorflow as tf
+import numpy as np
+import cv2
+import argparse
+
+def load_pb(path_to_pb):
+    with tf.io.gfile.GFile(path_to_pb, "rb") as f:
+        graph_def = tf.compat.v1.GraphDef()
+        graph_def.ParseFromString(f.read())
+    with tf.Graph().as_default() as graph:
+        tf.import_graph_def(graph_def, name='')
+        return graph
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("graph", help=".pb graph path")
```

Comment:
Can you please show how to run the script? Will `python3 demo_classification_tf.py --graph mygraph.pb` work? I mean `--`